README.md

```diff
@@ -1,56 +1,52 @@
-# O3DE-Extras
+# o3de-extras
 
-Extras add-on repo for O3DE.
+As the name implies, o3de-extras repo is for "extra" O3DE objects that are considered "canonical" to O3DE but not "core" to the engine.
+i.e. The core engine will build and run without the extras, however the o3de-extras contain extra projects, gems, etc. in a standard O3DE compound repo format.
```

Comment:
Do we have a standard compound repo folder structure that can be referenced in the user guide? If so, maybe we should link it?